### PR TITLE
feat(icons): add pen-off pencil-off, update pen icons w/ rounding

### LINKED ISSUE #2178: `pencil-off` icon
### Icon name

pencil-off

### Use cases

For a button that disables editing (Disable user edits) 

### Design ideas

the pencil icon with a diagonal line through it would be great! Similar to eye-off, nabigation-off etc

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

### PR DESCRIPTION
**Pull request by karsa-mistmere**
closes #2178

## What is the purpose of this pull request?

- [x] New Icon
- [x] Bug fix

### Description
Added `pen-off` and `pencil-off` as per #2178, but also updated pen/pencil icons to better match Lucide's style

### Icon use case 
see #2178

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] I've based them on the following Lucide icons: `pen` and `pencil`

### Naming 
- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by jguddas — CHANGES_REQUESTED** [review]
The end of the pen should be a half circle, it's more than 180deg right now.

**Comment by karsa-mistmere** [comment]
> The end of the pen should be a half circle, it's more than 180deg right now.

Can you elaborate, I don't think I follow.

**Comment by karsa-mistmere** [comment]
> > The end of the pen should be a half circle, it's more than 180deg right now.
> 
> Can you elaborate? I don't think I follow.

Ah, gotcha. Overzelous optimization at it again, I'll fix it.

**Review by jguddas — APPROVED** [review]

File: icons/clipboard-pen.svg [hunk 1]
```diff
@@ -10,7 +10,7 @@
   stroke-linejoin="round"
 >
   <rect width="8" height="4" x="8" y="2" rx="1" />
-  <path d="M10.4 12.6a2 2 0 0 1 3 3L8 21l-4 1 1-4Z" />
   <path d="M16 4h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2h-5.5" />
   <path d="M4 13.5V6a2 2 0 0 1 2-2h2" />
+  <path d="M13.378 15.626a1 1 0 1 0-3.004-3.004l-5.01 5.012a2 2 0 0 0-.506.854l-.837 2.87a.5.5 0 0 0 .62.62l2.87-.837a2 2 0 0 0 .854-.506z" />
```
[the remark this answers is not in the document]
Comment (karsa-mistmere) [reply icons/clipboard-pen.svg:15]: Yeah, definitely, it took me ages to get it to not go off grid. :D

**Pull request merged**